client/asset/xmr/process.go

```diff
@@ -138,11 +147,36 @@ func getRegtestWalletServerRpcPort(dataDir string) string {
 	return DefaultRegtestWalletServerRpcPort
 }
 
-func cliGenerateRefreshWallet(ctx context.Context, trustedDaemon string, net dex.Network, dataDir, cliToolsDir string, pw, seed []byte) error {
+// cliGenerateRefreshWallet generates a monero wallet from a given spendkey (seed)
+// and password bytes with a given birthday. Birthday should not be later than now.
+// Seed and password will be zeroed out on function exit.
+func cliGenerateRefreshWallet(
+	ctx context.Context,
+	trustedDaemon string,
+	log dex.Logger,
+	net dex.Network,
+	dataDir,
+	cliToolsDir string,
+	pw, seed []byte,
+	birthday uint64) error {
+
+	zero := func(bb []byte) {
+		for i := range len(bb) {
+			bb[i] = 0
+		}
+	}
+	defer func() {
+		zero(pw)
+		zero(seed)
+	}()
```

Comment:
Actually probably better the caller do this, but it doesn't really matter I guess.